src/main/java/com/dropbox/core/DbxSdkVersion.java

```diff
@@ -12,6 +12,6 @@
 
     private static String loadVersion()
     {
-        return "4.0.0"; //TODO: figure out a consistent way to generate it.
+        return "4.0.0-SNAPSHOT"; //TODO: figure out a consistent way to generate it.
```

Comment:
nit: can you remove the `TODO`. We aren't going to be tackling that any time soon, and if it ever becomes necessary, we won't need a `TODO` to tell us.

Reply:
Will do!